homeassistant/components/steamist/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""The Steamist integration."""
+from __future__ import annotations
+
+import logging
+
+from aiosteamist import Steamist
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+from .coordinator import SteamistDataUpdateCoordinator
+
+PLATFORMS: list[str] = [Platform.SWITCH]
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
`_LOGGER` here is defined but not used. Should we be aiming to standardise `LOGGER` as coming from `const.py`, defined against `__package__` instead of `__name__` and then imported when required?
`LOGGER = logging.getLogger(__package__)`
cc @frenck as that is the path he has chosen in the last few refactors I have seen.